react-native-libraries.json

```diff
@@ -8492,6 +8492,18 @@
     "web": true,
     "expo": true
   },
+  {
+    "githubUrl": "https://github.com/alabsi91/react-native-material-you-colors",
+    "examples": [
+      "https://github.com/alabsi91/react-native-material-you-colors/tree/main/example",
+      "https://snack.expo.dev/@alabsi91/react-native-material-you-colors"
+    ],
+    "newArchitecture": true,
```

Comment:
Hello @alabsi91, thank you for adding new entry to the directory! 👍 

One nit, and we can ship it:

```suggestion
```

We can skip this since `codegenConfig` is present in `package.json` and the flag will be set automatically.